### ISSUE TITLE
watch-storefront script issue

### ISSUE THREAD

**Issue by manubuer** [dockware/dockware#10]
hello there,

when I run the make watch-storefront command on an otherwise functioning contribute setup the following error occurs:

> ` ERROR  Failed to compile with 1 errors                                                                                                                               1:12:57 PM
> 
>  error  in /var/www/html/var/theme-entry.scss
> 
> Module build failed (from ../node_modules/sass-loader/lib/loader.js):
> 
>     a&:hover {
>     ^
>       Invalid CSS after "a": expected "{", was "&"
> 
> "&" may only be used at the beginning of a compound selector.
>       in /var/www/html/platform/src/Storefront/Resources/app/storefront/src/scss/skin/shopware/page/checkout/_cart-item.scss (line 38, column 6)
> 
>  @ /var/www/html/var/theme-entry.scss 2:14-347 21:1-42:3 22:19-352
>  @ multi ./src/main.js /var/www/html/custom/plugins/ManuTest/src/Resources/app/storefront/src/main.js /var/www/html/var/theme-entry.scss`

when commenting out the relevant line, the error not longer occurs and the compiling runs correctly. changes in the file are triggering the compilation, so the watcher must be active.

in both cases, though, just the html skeleton is being displayed as shown below.

<img width="444" alt="Screenshot 2020-09-10 at 13 47 15" src="https://user-images.githubusercontent.com/43177907/92725293-376b0700-f36c-11ea-8e17-2f57bdb8a275.png">

any ideas what I am missing here?
thanks

**Comment by dimitripreiss** [comment]
Hi manubuer,

what is youre Shopware version? I had a similar issue, after some hours I realized that I had updated Shopware to version 6.3.1.0 about the admin. After that `make watch-storefront` run without any error, but I also get the frontend with missed JS and CSS files (404).

I guess, at the moment Dockware supports only Shopware up to 6.3.0. Because there is no variant for 6.3.1 in `dockware/variants/dev/`.  

@dockware is that’s right?
